app/general/admin.py

```diff
@@ -47,6 +47,7 @@ class DocumentFileAdmin(SimpleHistoryAdmin):
     ordering = ["title"]
     list_display = ["title", "license", "document_type", "available"]
     search_fields = ["title", "license", "document_type"]
```

Comment:
Please limit searching to title only.

Reply:
@friedelwolff  I will get Daniel to look into this for you, thanks

Reply:
@OnaMosimege,  please would you approve the changes made, thank you

Reply:
@friedelwolff I approve, please check if you're 100% with everything?